src/mock_runtime_test.cc

```diff
@@ -129,9 +129,9 @@ TEST(MockRuntimeTest, JSSendArrayBufferViewTypes) {
   Deno* d = deno_new(nullptr, [](auto _, auto buf) {
     count++;
     size_t data_offset = buf.data_ptr - buf.alloc_ptr;
-    EXPECT_EQ(data_offset, 2468);
-    EXPECT_EQ(buf.data_len, 1000);
-    EXPECT_EQ(buf.alloc_len, 4321);
+    EXPECT_EQ(data_offset, 2468u);
```

Comment:
It's surprising we are getting different warnings on different platforms since we're supposedly sharing a clang compiler and libc++... No, I don't have anything custom. This is using this build configuration:
```
gn gen out/Debug --args='cc_wrapper="ccache" is_debug=true v8_optimized_debug=false'
```